### ISSUE TITLE
ReferenceUsedNamesOnly does not detect references without leading "\" namespace separator

### ISSUE THREAD

**Issue by klausi** [slevomat/coding-standard#1570]
Hi, not sure if this is a bug or intended behavior. Give this test file:
```php
<?php

$x = new Vendor\DateTools\Erm();
```
and executing
```
./vendor/bin/phpcbf --standard=SlevomatCodingStandard --sniffs=SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly Test.php
```
I would expect this to be fixed to
```php
<?php
use Vendor\DateTools\Erm;

$x = new Erm();
```

Am I missing something why this does not work?

Background: trying to replace Drupal Coder's custom use statement sniff with your upstream code to avoid maintaining it in https://github.com/pfrenssen/coder/pull/199

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/019d946f1babedb9cfa8eaddec29cddea371e579

**Comment by klausi** [comment]
Oh wow - thanks a lot for the quick fix!
